app/models/article.rb

```diff
@@ -317,7 +317,7 @@ def readable_edit_date
   end
 
   def readable_publish_date
```

Comment:
**thought (non-blocking):** Looking at this it seems to be a display concern, so in the spirit of cleaning up our models a bit, could it make sense to move this (and the new `displayable_published_at` method) into a decorator?